crm_location_nuts/models/crm_lead.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    region = fields.Many2one(comodel_name='res.partner.nuts', string="Region")
```

Comment:
@rafaelbn why this is called this way and there are only 2 levels instead of four as in base_location_nuts?

Reply:
@rafaelbn Can you attend this comment?

Reply:
@chienandalu because that 2 levels are the only two relevant for address purpose

Reply:
@chienandalu @pedrobaeza testing this in runbot it doesn't work. So I think it must be imlpemented as in `base_location_nuts`. Could you fix it ? Thanks and sorry for the answer I must test it before.